CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## master
+
+* Adds env variable ```INSTABUG_AUTO_UPLOAD_ENABLE_IOS``` to support skipping iOS source maps auto upload
```

Comment:
I think you can replace \`\`\` surrounding the env var name with \` since \`\`\` is primarily used for multi-line code and \` is for inline code.